docs/03-04-function-configuration-file.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Function's configuration file
+type: Details
+---
+
+When you initialize a Function (`init`), CLI creates the `config.yaml` file in your workspace folder. This file contains the whole Function's configuration and specification not only for the Function custom resource but also any other related resources you create for it, such as Subscriptions and API Rules.
```

Comment:
```suggestion
When you initialize a Function (with the `init` command), CLI creates the `config.yaml` file in your workspace folder. This file contains the whole Function's configuration and specification not only for the Function custom resource but also any other related resources you create for it, such as Subscriptions and API Rules.
```

If the user didn't come directly from the other doc where the `init` command is described, maybe they wouldn't know what exactly is being referred to? But again, this is just a suggestion.

Reply:
You're right, thx :)